lib/twitter-ads/version.rb

```diff
@@ -1,5 +1,5 @@
 # Copyright (C) 2015 Twitter, Inc.
 
 module TwitterAds
-  VERSION = '0.3.1'
+  VERSION = '0.3.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.